staggeredanimationgroup/src/test/java/com/bartoszlipinski/constraint/StaggeredAnimationGroupTest.java

```diff
@@ -34,7 +34,7 @@
     public void filterNonZeroIds_returns_shorterArray_whenThereAre_zeros_inPassedArray() {
         //given
         StaggeredAnimationGroup spiedGroup = prepareSpiedGroup();
-        int[] testIds = new int[]{1, 2, 3, 4, 0, 0, 0, 0};
+        int[] testIds = new int[]{1, 2, 0, 3, 4, 0, 0, 0, 0};
```

Comment:
When can this happen?
`ConstraintLayout` was using trailing zeros if there were "not enough" ids to fill the whole array. Did this change?

Reply:
No problem, thank you for taking the time to analyse things!